src/validation.js

```diff
@@ -0,0 +1,29 @@
+const COMMENT_LENGTH = {
+  MAX: 300,
+  MIN: 50
+};
+
+const ratingValidation = (ratingNumber) => {
+  if (ratingNumber <= 0) {
```

Comment:
Как-то странновато выглядит эта проверка, не? Не очень согласуется с бизнес-логикой. Как поставить рейтинг ниже нуля?)
Кажется, что достаточно такого:
```suggestion
  if (!ratingNumber) {
```